C7GameData/Save/SavePlayer.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+
+namespace C7GameData.Save {
+
+	public class SavePlayer {
+		public ID id;
+		public uint color;
+		public bool barbarian;
+		public bool human = false;
+		public bool hasPlayedCurrentTurn = false;
```

Comment:
Why is this variable necessary/how is the variable intended to be used? Wouldn't the game just see if the player has unfortified/unskipped units with movement left and, if not, just allow the player to end turn?

Reply:
hasPlayedCurrentTurn may be true when a player has remaining movement points but ends their turn early, which I don't think can be inferred from other state and needs to be stored directly

Reply:
I see, such as when hitting "shift+enter"